docs/pages/getting-started/linux-server.mdx

```diff
@@ -20,52 +20,128 @@ Teleport (=teleport.version=) on Linux machines.
 (!docs/pages/includes/permission-warning.mdx!)
 
 <Tabs>
-  <TabItem label="Amazon Linux 2/RHEL (RPM)">
+  <TabItem label="Community">
```

Comment:
@stevenGravy let's wait a bit until Enterprise top level switcher will be ready, so we don't have to introduce another layer. It will be ready next week and we can start porting content to it

Reply:
@klizhentas ok thanks, will update then!

Reply:
@klizhentas I have updated to use the OpenSource and Enterprise scopes. I kept the layers since you can't hide/show `Tabs` based on the scope. Here's what it looks like. 
![image](https://user-images.githubusercontent.com/60704961/136028203-e1e7a9b5-20ab-44e2-be80-252d092308ac.png)